<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>301</width>
     <height>197</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QPushButton" name="spareSpace_PushButton">
      <property name="text">
       <string>空余车位：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="waitingCars_PushButton">
      <property name="text">
       <string>等候车辆：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="entryButton">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>进车</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="outButton">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>出车</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="infoButton">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>历史信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="income_pushButton">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 164, 119);
color: rgb(0, 0, 0);
font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>总收入</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="currentCars_PushButton">
      <property name="text">
       <string>当前车辆：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="numberOfAll_PushButton">
      <property name="text">
       <string>车位总数：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="5">
     <widget class="QTextBrowser" name="info_textBrowser"/>
    </item>
    <item row="3" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
